containers/login/LoginContainer.js

```diff
@@ -9,16 +9,6 @@ import TOTPForm from './TOTPForm';
 import UnlockForm from './UnlockForm';
 import useLogin from './useLogin';
 
-const getErrorText = (error) => {
-    if (error.name === 'PasswordError') {
-        return c('Error').t`Incorrect decryption password`;
```

Comment:
This one is ok since it's in a function

Reply:
Don't know why I moved it inside... :man_shrugging: